frontend/src/queries/examples.ts

```diff
@@ -35,73 +51,78 @@ const LegacyTrendsQuery: LegacyQuery = {
+                    key: '$current_url',
                     operator: PropertyOperator.Exact,
-                    value: 'Chrome',
+                    value: ['https://hedgebox.net/files/'],
                 },
-                {
-                    type: PropertyFilterType.Cohort,
-                    key: 'id',
-                    value: 2,
-                },
-            ],
-            limit: 100, // TODO - can't find a use for `limits` in insights/trends
-        },
-        {
-            kind: NodeKind.ActionsNode,
-            id: 1,
-            name: 'Interacted with file',
-            custom_name: 'Interactions',
-            properties: [
                 {
                     type: PropertyFilterType.Event,
                     key: '$geoip_country_code',
                     operator: PropertyOperator.Exact,
-                    value: ['US'],
+                    value: ['US', 'AU'],
                 },
             ],
-            math: PropertyMathType.Average,
-            math_property: '$session_duration',
         },
     ],
-    filterTestAccounts: false,
-    properties: {
-        type: FilterLogicalOperator.And,
-        values: [
+}
+
+const filterTestAccounts = false
+const series: (EventsNode | ActionsNode)[] = [
+    {
+        kind: NodeKind.EventsNode,
+        name: '$pageview',
+        custom_name: 'Views',
+        event: '$pageview',
+        properties: [
             {
-                type: FilterLogicalOperator.Or,
-                values: [
-                    {
-                        type: PropertyFilterType.Event,
-                        key: '$current_url',
-                        operator: PropertyOperator.Exact,
-                        value: ['https://hedgebox.net/files/'],
-                    },
-                    {
-                        type: PropertyFilterType.Event,
-                        key: '$geoip_country_code',
-                        operator: PropertyOperator.Exact,
-                        value: ['US', 'AU'],
-                    },
-                ],
+                type: PropertyFilterType.Event,
+                key: '$browser',
+                operator: PropertyOperator.Exact,
+                value: 'Chrome',
+            },
+            {
+                type: PropertyFilterType.Cohort,
+                key: 'id',
+                value: 2,
             },
         ],
+        limit: 100, // TODO - can't find a use for `limits` in insights/trends
+    },
+    {
+        kind: NodeKind.ActionsNode,
+        id: 1,
+        name: 'Interacted with file',
+        custom_name: 'Interactions',
+        properties: [
+            {
+                type: PropertyFilterType.Event,
+                key: '$geoip_country_code',
+                operator: PropertyOperator.Exact,
+                value: ['US'],
+            },
+        ],
+        math: PropertyMathType.Average,
+        math_property: '$session_duration',
+    },
```

Comment:
I don't have an action with this ID locally, so most of these example queries show nothing for me. I'm not sure if we should have queries that work for anyone or not. Fine to keep as is for now 🤷